src/assets/icons/UserAdd.tsx

```diff
@@ -1,6 +1,6 @@
 import IconProps from '../iconinterface'
 
-export default function UserAddIcon({ className }: IconProps) {
+export default function UserAddIcon({ className, onClick }: IconProps) {
```

Comment:
If this icon is used inside a button, let's give the onClick event to the button instead of this icon. Since this is a re-usable icon, its best to make it as independent as possible.